media-video/cheese/cheese-3.34.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome.org vala virtualx meson
+
+DESCRIPTION="A cheesy program to take pictures and videos from your webcam"
+HOMEPAGE="https://wiki.gnome.org/Apps/Cheese"
+
+LICENSE="GPL-2+"
+SLOT="0/8" # subslot = libcheese soname version
+IUSE="gtk-doc +introspection test X"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+REQUIRED_USE="test? ( X )"
+DEPEND="
+	>=dev-libs/glib-2.39.90:2
+	>=x11-libs/gtk+-3.13.4:3[introspection?]
+	>=gnome-base/gnome-desktop-2.91.6:3=
+	>=media-libs/libcanberra-0.26[gtk3]
+	>=media-libs/clutter-1.13.2:1.0[introspection?]
+	>=media-libs/clutter-gtk-0.91.8:1.0
+	media-libs/clutter-gst:3.0
+	media-libs/cogl:1.0=[introspection?]
+
+	media-video/gnome-video-effects
+	x11-libs/gdk-pixbuf:2[jpeg,introspection?]
+
+	>=media-libs/gstreamer-1.4:1.0[introspection?]
+	>=media-libs/gst-plugins-base-1.4:1.0[introspection?,ogg,pango,theora,vorbis]
+
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+	X? (
+		>=media-libs/gst-plugins-base-1.4:1.0[X]
+		x11-libs/libX11
+		x11-libs/libXtst
```

Comment:
This is only used by test programs, so should go inside `test? ( x11-libs/libXtst )` inside `DEPEND`.